frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -2282,9 +2282,9 @@ var AppRouter = Backbone.Router.extend({
                         });
                         exportDrop.on("open",function(){ 
                             if(exportAPIData) {
-                                $(".server-export .countly-drop-content").empty().append(CountlyHelpers.export(oSettings._iRecordsDisplay, exportAPIData, null, true).removeClass("dialog"));
+                                $(".server-export .countly-drop-content").empty().append(CountlyHelpers.export(oSettings._iRecordsDisplay, app[exportView].getExportAPI(oSettings.sTableId), null, true).removeClass("dialog"));
                             } else if(exportQueryData) {
-                                $(".server-export .countly-drop-content").empty().append(CountlyHelpers.export(oSettings._iRecordsDisplay, exportQueryData).removeClass("dialog"));
+                                $(".server-export .countly-drop-content").empty().append(CountlyHelpers.export(oSettings._iRecordsDisplay, app[exportView].getExportQuery(oSettings.sTableId)).removeClass("dialog"));
```

Comment:
What would be the difference between these two? Either stored in variable or direct method call?

Reply:
Because this function triggered by user click download icon to show download menu.
before that time, user may change the filter condition for talble, So we had better reload the newest query string for download.